script.js

```diff
@@ -43,22 +43,45 @@ $(document).ready(function () {
 
     //create ships
     var hitCount = 0;
+    var col, row;
+    var numberShips = 5;
 
     var gameBoardWithShips = [
-        [0, 0, 0, 1, 0, 0, 0],
-        [0, 0, 1, 0, 0, 0, 0],
         [0, 0, 0, 0, 0, 0, 0],
-        [0, 1, 0, 0, 0, 0, 0],
         [0, 0, 0, 0, 0, 0, 0],
-        [0, 0, 0, 0, 0, 1, 0],
-        [1, 0, 0, 0, 0, 0, 0]
-    ]
+        [0, 0, 0, 0, 0, 0, 0],
+        [0, 0, 0, 0, 0, 0, 0],
+        [0, 0, 0, 0, 0, 0, 0],
+        [0, 0, 0, 0, 0, 0, 0],
+        [0, 0, 0, 0, 0, 0, 0]
+    ];
+
+    function insertShip() {
+
+        row = Math.floor(Math.random() * rowsCount);
+        col = Math.floor(Math.random() * colsCount);
+        if (gameBoardWithShips[row][col] !== 1) {
+
+            gameBoardWithShips[row][col] = 1;
+        } else {
+            insertShip()
```

Comment:
No `;` at end of the line. And also in some specific situation you can get stack overflow here.